### PR TITLE
Reorganise create peer API docs, add list scripts and kafka peer

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
This PR:
- Splits the existing create peer doc into docs for each peer.
- Adds doc for creating kafka peer
- Adds doc for listing scripts
- Displays banners regarding peerdb cloud availability 

Screenshot 2024-10-17 at 6 40 47 PM
<hr/>
Screenshot 2024-10-17 at 6 41 18 PM

### CODE REVIEW

File: peerdb-api/endpoints/create-peer/kafka.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Create a Kafka peer"
+---
+You can create a Kafka peer using this endpoint. Here is the request and response format for creating a Kafka peer.
+
+<Info> This peer is only supported in [PeerDB OSS](https://github.com/PeerDB-io/peerdb) and [PeerDB Enterprise](https://github.com/PeerDB-io/peerdb-enterprise) </Info>
```
Comment (iamKunalGupta) [inline peerdb-api/endpoints/create-peer/kafka.mdx:6]: I don't think this is required as PeerDB Cloud has a banner mentioning only CH support

File: peerdb-api/endpoints/create-peer/clickhouse.mdx [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-## Postgres Peer
-Here is the request and response format for creating a Postgres peer. SSH tunneling is optional for peer creation.
-
+---
+title: "Create a ClickHouse peer"
+---
+You can create a Clickhouse peer using this endpoint. Here is the request and response format for creating a Clickhouse peer.
+Note that the Clickhouse peer uses an intermediary S3 stage under the hood for performance. The S3 fields related to this are optional, because:
+- For PeerDB Cloud, the S3 bucket is managed by PeerDB
+- For PeerDB OSS, a minio bucket is provided as part of the stack.
```
Comment (iamKunalGupta) [inline peerdb-api/endpoints/create-peer/clickhouse.mdx:7]: Move to `<Info>` or `<Note>` ?

File: peerdb-api/endpoints/list-scripts.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+
+```http
+GET /api/v1/scripts/{id}
+```
+
+This endpoint is used to get a script by its ID. If the ID is -1, it returns all scripts.
+
+<Info>
+Scripts by default is disabled in [PeerDB Cloud](https://auth.peerdb.cloud/en/login).
```
Comment (iamKunalGupta) [inline peerdb-api/endpoints/list-scripts.mdx:9]:
```suggestion
Scripts are disabled by default in [PeerDB Cloud](https://auth.peerdb.cloud/en/login).
```

File: peerdb-api/endpoints/create-peer/clickhouse.mdx (hunk 2 above)
Comment (heavycrystal) [inline peerdb-api/endpoints/create-peer/clickhouse.mdx:9]: There is a gotcha here, OSS minio needs to be exposed if CH isn't on the same machine

File: peerdb-api/endpoints/create-peer/kafka.mdx (hunk 1 above)
Comment (heavycrystal) [reply peerdb-api/endpoints/create-peer/kafka.mdx]: it wouldn't be clear if a user goes to the instance directly right?

File: peerdb-api/endpoints/create-peer/clickhouse.mdx (hunk 2 above)
Comment (Amogh-Bharadwaj) [reply peerdb-api/endpoints/create-peer/clickhouse.mdx:9]: I don't think we need to go in-depth into that in the API doc
Should do it in the setup doc

**Review by heavycrystal — APPROVED** [review]

**Pull request merged**